src/bin/pgcopydb/ld_stream.c

```diff
@@ -35,6 +35,150 @@
 static bool updateStreamCounters(StreamContext *context,
 								 LogicalMessageMetadata *metadata);
 
+/*
+ * Determines whether a message needs to be filtered out based on the
+ * provided filters. The caller function should have already parsed and
+ * validated the filters.
+ */
+bool
+ShouldFilterOutMessage(SourceFilters *filters, char *nspname, char *relname)
+{
+	/*
+	 * Validate nspname is not NULL or empty
+	 */
+	if (nspname == NULL || IS_EMPTY_STRING_BUFFER(nspname))
+	{
+		log_error("BUG: nspname is NULL or empty");
+		return false;
+	}
+
+	/*
+	 * Validate relname is not NULL or empty
+	 */
+	if (relname == NULL || IS_EMPTY_STRING_BUFFER(relname))
+	{
+		log_error("BUG: relname is NULL or empty");
+		return false;
+	}
+
+	/*
+	 * Filter-out pgcopydb.* in any case
+	 */
+	if (streq(nspname, "pgcopydb"))
+	{
+		log_info("Filtering out message for pgcopydb internal tables \"%s\"", nspname);
+		return true;
+	}
```

Comment:
Now that the sentinel table is created in the SQLite database catalogs, rather than on the source server, do we still have `pgcopydb.*` tables to filter out at all?

Reply:
[0e4dc40](https://github.com/dimitri/pgcopydb/pull/614/commits/0e4dc403e4790c38da8173b042d652ee8fab0f48)